pkgs/applications/science/misc/netlogo/default.nix

```
@@ -0,0 +1,43 @@
+{ jre, stdenv, fetchurl, makeWrapper }:
+
+
+stdenv.mkDerivation rec {
+  name = "NetLogo-${version}";
+  version = "6.0.4";
+
+
+  src = fetchurl {
+    url = "https://ccl.northwestern.edu/netlogo/6.0.4/NetLogo-6.0.4-64.tgz";
+    sha256 = "0dcd9df4dfb218826a74f9df42163fa588908a1dfe58864106936f8dfb76acec";
+  };
+
+
+  buildInputs = [ makeWrapper ];
+
+
+  installPhase = ''
+    mkdir -pv $out/share/netlogo
+    cp -rv app readme.md $out/share/netlogo
+
+    # launcher with `cd` is required b/c otherwise the model library isn't usable
+    launcher="$out/bin/netlogo"
+    mkdir -p $out/bin
+    echo "#!/bin/sh" > $launcher
+    echo "cd $out/share/netlogo/app" >> $launcher
+    echo "exec ${jre}/bin/java -jar netlogo-6.0.4.jar" >> $launcher
+    chmod +x $out/bin/netlogo
+  '';
+
+
+  meta = with stdenv.lib; {
+    description = "A multi-agent programmable modeling environment";
+    longDescription = ''
+      NetLogo is a multi-agent programmable modeling environment. It is used by
+      many tens of thousands of students, teachers and researchers worldwide.
+    '';
+    homepage = https://ccl.northwestern.edu/netlogo/index.shtml;
+    license = licenses.gpl2;
+    maintainers = [ maintainers.dpaetzel ];
+    platforms = stdenv.lib.platforms.linux;
```

```suggestion
    platforms = platforms.linux;
```

Since we have `with stdenv.lib`

---

Yup, my mistake. Thank you!